tune_sklearn/tune_basesearch.py

```diff
@@ -735,18 +739,37 @@ def _fill_config_hyperparam(self, config):
         """
         raise NotImplementedError("Define in child class")
 
-    def _tune_run(self, config, resources_per_trial):
+    def _tune_run(self, config, resources_per_trial, tune_params=None):
         """Wrapper to call ``tune.run``. Implement this in a child class.
 
         Args:
             config (:obj:`dict`): dictionary to be passed in as the
                 configuration for `tune.run`.
             resources_per_trial (:obj:`dict` of int): dictionary specifying the
                 number of cpu's and gpu's to use to train the model.
+            tune_params (dict, optional): User defined parameters passed to
+                ``tune.run``.
 
         """
         raise NotImplementedError("Define in child class")
 
+    def _override_run_args_with_tune_params(self, run_args, tune_params):
+        """Helper method to override tune.run run arguments with user
+        supplied dict"""
```

Comment:
nit (not important here but for future ref), the first line of the docstring is a summary line - and should not have a linebreak in the middle. this is important when building docs